cmd/server/main.go

```diff
@@ -141,6 +184,32 @@ func handleAndroidVariant(key string, clientId string, pn string) {
 	}
 }
 
+func handleIOSVariant(clientId string, cert string, passPhrase string) {
+	// Only instantiate the push client here because we need to wait for the ups secret to
+	// be available
+	if pushClient == nil {
+		pushClient = pushClientOrDie()
+	}
+	certByteArray := []byte (cert)
+	payload := &iOSVariant{
```

Comment:
Ignoring duplicate check which happens for android for now

Reply:
@matzew Should we check for duplicate variants or can it make sense to have variants with the same name/credentials? For Android it's easy to discover duplicates because UPS returns the google key. How could we do that for iOS other than using the name?

Reply:
@pb82 @matzew  Currently the UPS UI will allow two iOS variants created with the same name and credentials. Obviously the variant ids will be different once created